### DISCUSSION TITLE
【V5】子刻度线

### DISCUSSION THREAD

**Discussion by BinghuiXie** [antvis/G2#5407] [Q&A]
- **G2 Version**: latest
- **Platform**:
- **Mini Showcase(like screenshots)**:
- **CodePen Link**:

V4 中子刻度线（`subTickLine`）的概念，看了下 V5 的代码，好像并没有相关的实现，是在 V5 中已经抛弃这个概念了吗？

**Comment by hustcc** [comment]
是的，v5 取消 sub tick 的概念了，你是在什么场景上，对于 sub tick 诉求很强吗？

**Reply by BinghuiXie** [reply]
之前基于 V4 封装了一层 React 组件库使用，现在想跟着 V5 一起把底层升级掉。所以想尽量保证上层的改动较小。
顺便了解下，sub tick 的概念为什么要取消呢？
